src/extensiveFormulation.jl

```diff
@@ -7,47 +7,46 @@
 #############################################################################
 
 
-function extensive_formulation(model, 
+function extensive_formulation(model,
                                params)
 
-    
+
     #TODO Recover all the constant in the model or in param
     laws = model.noises
     N_NOISES = laws[1].supportSize
-    
+
     DIM_STATE = model.dimStates
     DIM_CONTROL = model.dimControls
-    
+
     X_init = model.initialState
-    
-    
+
+
     T = model.stageNumber-1
-    
+
     mod = Model(solver=params.solver)
-    
-    
+
+
     #Calculate the number of nodes n at each step on the scenario tree
     N = Array{Int64,2}(zeros(T+1,1))
     N[1] = 1
     for t = 1 : (T)
         N[t+1] = N[t]*laws[t].supportSize
     end
-    
-    
-    
+
+
+
     #Define the variables for the extensive formulation
     #At each node, we have as many variables as nodes
     @defVar(mod,  u[t=1:T,n=1:DIM_CONTROL*N[t+1]])
     @defVar(mod,  x[t=1:T+1,n=1:DIM_STATE*N[t]])
     @defVar(mod,  c[t=1:T,n=1:N_NOISES*N[t]])
-    
-    
+
+
     #Define the conditional probabilities on each arc of the scenario tree
```

Comment:
No, it computes the total probability of each node from the conditional probabilities.